IDE/STM32CUBE/default_conf.ftl

```diff
@@ -68,24 +68,80 @@ extern ${variable.value} ${variable.name};
 #define NO_MAIN_DRIVER
 #define WOLFTPM_EXAMPLE_HAL
 
+/* Set smaller default timeout for embedded devices */
+#define TPM_TIMEOUT_TRIES 10000
+
+/* Example for TPM wait delay */
+#if 0
+    #define XTPM_WAIT() HAL_Delay(1)
+#endif
+
 /* ------------------------------------------------------------------------- */
 /* Enable Features */
 /* ------------------------------------------------------------------------- */
 #undef WOLFTPM2_NO_WOLFCRYPT
 #if defined(WOLFTPM_CONF_WOLFCRYPT) && WOLFTPM_CONF_WOLFCRYPT == 0
     #define WOLFTPM2_NO_WOLFCRYPT
 #endif
+#ifndef WOLFTPM2_NO_WOLFCRYPT
+    /* pull in wolfCrypt build settings */
+    #include "wolfSSL.I-CUBE-wolfSSL_conf.h"
```

Comment:
This should not be necessary as `wolfssl/wolfcrypt/settings.h` should pull in `wolfSSL.I-CUBE-wolfSSL_conf.h`

Reply:
You are correct. I thought this was needed for the customer project, but it is not. Removed.